Xamarin.Essentials/FilePicker/FilePicker.android.cs

```diff
@@ -86,5 +86,11 @@ public partial class FilePickerFileType
             {
                 { DevicePlatform.Android, new[] { "video/*" } }
             });
+
+        public static FilePickerFileType PlatformPdfFileType() =>
```

Comment:
Could we make these internal? For all the platforms. I know that you just copied the surrounding code, but I think this should have not been public since the correct access is the public properties.

Reply:
Great suggestion, I've made the method `private` instead of `internal` as it is only used in the `partial` class.